### PR TITLE
Portfolio homepage done

### PR DESCRIPTION
**Pull request by NicolasAndrehh**
# Milestone - Homepage completed
In this pull request I implement the following features:
- Added professional `README.md` file
- Added `navbar` and main content on the html file
- Added `social media` links
- Added `Media Query` styles for mobile version

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    display: flex;
+    max-width: 100vw;
+    min-height: 100vh;
+    background-image: url(header-illsutration-mobile@2x.png);
+    background-size: 100vw;
+    border-bottom-left-radius: 2rem;
+    flex-direction: column;
+    padding: 2rem;
+    justify-content: space-around;
+  }
```
Comment (RaoAkif) [inline styles.css]: image

The background image is repeating here. You can add `background-repeat: no-repeat;` to stop the repetition of the background.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+</head>
+<body>
+    <div class="wrapped">
+        <nav class="navbar">
+            <h2>Welcome</h2>
+            <a href="#" class="options"><i class="las la-bars"></i></a>
+            <a href="#" class="shortcuts">Portfolio</a>
+            <a href="#" class="shortcuts">About</a>
+            <a href="#" class="shortcuts">Contact</a>
```
Comment (RaoAkif) [inline index.html]: To write semantic code, when you see that there is a list of links or items in the HTML code, you can use `ul` tag with `li` elements.
Then place these `a` tags inside each `li` tag.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    font-size: 2.5rem;
+    text-align: center;
+  }
+
+  #homepage h1 span {
+    color: rgb(45, 204, 151);
+  }
+
+  #homepage p {
+    font-size: larger;
```
Comment (RaoAkif) [inline styles.css]: Kindly try to follow Figma design, font, styles, etc. as much as you can.
- As per Figma design, this `font-size` is `20px`
- Also add `line-height` to the paragraph, which is missing here.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                <li class="social">
+                    <a href="https://github.com/NicolasAndrehh"><i class="lab la-github"></i></a>
+                </li>
+                <li class="social">
+                    <a href="https://www.linkedin.com/in/nicolas-andres-olaya-gamba-3b032b248/"><i class="lab la-linkedin-in"></i></a>
+                </li>
+                <li class="social">
+                    <a href="https://twitter.com/nicolasolaya22"><i class="lab la-twitter"></i></a>
+                </li>
+            </ul>
```
Comment (RaoAkif) [inline index.html]: Please try to follow the Figma design.
image

This is yours:
image

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @NicolasAndrehh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To keep the project organized, It would be good to create a folder named let say `images` and put your images there. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NicolasAndrehh** [comment]
# Hello @RaoAkif!
Thank you so much for your feedback it helps me a lot, I already do all the changes that you suggest, do you think I do it good?

**Comment by RaoAkif** [comment]
> # Hello @RaoAkif!
> Thank you so much for your feedback it helps me a lot, I already do all the changes that you suggest, do you think I do it good?

Yeah @NicolasAndrehh, a  lot better 👌🏻

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+  #homepage h1 {
+    color: rgb(0, 0, 88);
+    font-size: 2.5rem;
+    text-align: center;
+    font-weight: 700;
+  }
+
+  #homepage h1 span {
+    color: rgb(45, 204, 151);
```
Comment (huseyinbiyik) [inline styles.css:72]: [OPTIONAL] 
Please stick to the design and follow color code on the Figma templete.
Screenshot 2023-01-12 at 11 51 00 PM

**Review by huseyinbiyik — APPROVED** [review]
Hi @NicolasAndrehh,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Good job implementing the requested changes 👨‍🎨
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 5 above)
Comment (NicolasAndrehh) [reply styles.css:72]: Okay, I'll do it, thanks for the feedback! :D

**Pull request merged**